src/docs/product/accounts/pricing.mdx

```diff
@@ -295,9 +295,13 @@ You will be downgraded to the same plan you were on prior to starting the trial.
 
 You control who can see events. We allow you to add your team members, as well as share events with anonymous users should you wish to do so.
 
-#### Events now include transactions, errors, and attachments? What's the difference?
+#### Events now include transactions and errors. What's the difference? {#events-now-include-transactions-errors-and-attachments-whats-the-difference}
 
-An _event_ is one instance of you sending Sentry data. What can be captured as an error depends on your platform. In general, if you have something that looks like an exception, it will be sent to Sentry as an error event. _Attachments_ are files uploaded in the same request, such as log files. In addition, the files within the crash uploaded to Sentry can be persisted by enabling storage of crash reports. A _transaction_ represents a single instance of a service being called to support an operation you want to measure or track, like a page load.
+An _event_ is one instance of you sending Sentry data. What can be captured as an error depends on your platform. In general, if you have something that looks like an exception, it will be sent to Sentry as an _error event_. A _transaction event_ represents a single instance of a service being called to support an operation you want to measure or track, like a page load.
```

Comment:
>An _event_ is one instance of you sending Sentry data.

This is incorrect, and I don't know how to word this without referencing errors, transactions, or attachments. Maybe:

> An _event_ is one instance of you sending Sentry data, excluding attachments.

Reply:
One instance of sending data is a request someone makes. You can make a request and only send an attachment. That's an instance of sending data that isn't an event.

Reply:
Ah okay, I understand.

Reply:
```suggestion
An _event_ is one instance of you sending Sentry data, excluding attachments. What can be captured as an error depends on your platform. In general, if you have something that looks like an exception, it will be sent to Sentry as an _error event_. A _transaction event_ represents a single instance of a service being called to support an operation you want to measure or track, like a page load.
```

Reply:
Looks good to me.